### ISSUE TITLE
An error in halMergeChroms

### ISSUE THREAD

**Issue by zhangqing970** [ComparativeGenomicsToolkit/cactus#1168]
Hi, 
          I was run the cactus by split genome into multiple chromosomes, now i want to merge the HAL file, when i run halMergeChroms, i got this error:

   terminate called after throwing an instance of 'hal_exception'
  what():  Duplicate sequence name found: Anc01.Anc01refChr0
/var/spool/slurm/d/job9356440/slurm_script: line 9:  8590 Aborted                 (core dumped) halMergeChroms Cactus_Chr01.hal,Cactus_Chr02.hal,Cactus_Chr03.hal,Cactus_Chr04.hal,Cactus_Chr05.hal,Cactus_Chr06.hal,Cactus_Chr07.hal,Cactus_Chr08.hal,Cactus_Chr09.hal,Cactus_Chr10.hal,Cactus_Chr11.hal,Cactus_Chr12.hal,Cactus_Chr13.hal,Cactus_Chr14.hal,Cactus_Chr15.hal Cactus.hal

Can you give me advise on how to fix this error ?

**Comment by zhangqing970** [comment]
> `halMergeChroms` only runs on star trees (ie those used in pangenome alignments). It does not work with Progressive Cactus alignments.

Is there any way to merge the HALs ?  because the multiple genome alignment is too time cost.

**Comment by yaoxkkkkk** [comment]
Hi, I also met same error with you, do you solve the problem?

**Comment by yaoxkkkkk** [comment]
I manually extracted chromosomes (e.g. all chr01) from each assembly and run `cactus-pangenome` on each chromosome community, thus I obtained several graph files for every chromosome. Now I want to merge them together by `cactus-graphmap-join`, the error came up in `halMergeChroms` step:
```bash
	terminate called after throwing an instance of 'hal_exception'
	  what():  Duplicate sequence name found: _MINIGRAPH_.s37013
```

**Comment by yaoxkkkkk** [comment]
> You might be able to hack around this by renaming MINIGRAPH paths in your input so that the names are unique.

Yes I would like to know how to manipulate these graph files, could you please give me some advice on command and software used?

**Comment by glennhickey** [comment]
For hal files (which you don't need to pass to cactus-grpahmap-join):  `halRenameSequences`
For vg files,  use `vg convert -f` to convert to gfa, then a script/text editor (`sed`?), then convert back with `vg convert -p`.

**Comment by yaoxkkkkk** [comment]
Thank you very much about that!
